proto/buf.gen.gogo.yaml

```
@@ -0,0 +1,8 @@
+version: v1
+plugins:
+  - name: gocosmos
+    out: ..
+    opt: plugins=interfacetype+grpc,Mgoogle/protobuf/any.proto=github.com/celestiaorg/celestia-app/codec/types
+  - name: grpc-gateway
+    out: ..
+    opt: logtostderr=true,allow_colon_final_segments=true
```

Was this generated automatically? If not, I would prefer if we end a line at  the end of the file:

```suggestion
    opt: logtostderr=true,allow_colon_final_segments=true
    
```